concordium-node/src/p2p/connectivity.rs

```diff
@@ -564,6 +564,7 @@ pub fn connection_housekeeping(node: &Arc<P2PNode>) -> bool {
 
     let curr_stamp = get_current_stamp();
     let peer_type = node.peer_type();
+    let max_keep_alive_ms = node.config.max_normal_keep_alive * 1000;
```

Comment:
Why not store it in ms. instead of doing the ms -> sec conversion for every housekeeping.

Reply:
Well.. true, could do the conversion when creating the node config, however I thought it was nice for the user to input in seconds when configuring the node.

For now I'll put the conversion in the node config.

Reply:
Yes, I still think the node should be configured with secs.